Exiled.Events/Patches/Events/Warhead/Detonated.cs

```diff
@@ -36,7 +36,7 @@ private static IEnumerable<CodeInstruction> Transpiler(IEnumerable<CodeInstructi
             newInstructions[0].labels.Add(continueLabel);
 
             newInstructions.InsertRange(
-                0,
+                newInstructions.Count - 1,
```

Comment:
This is detonating event and it's calling before warhead detonate

Reply:
Yeah, i see, someone fixed it but i found another problem